app/views/jobs/_job_card.html.erb

```diff
@@ -4,6 +4,13 @@
     <p>Job Link: <%= job.link %></p>
     <p>Job Company: <%= job.company %></p>
     <p>Job Location: <%= job.location %></p>
+    <%= form_with model: job do |form| %>
+      <div>
+        <%= form.label :status %>
+        <%= form.select :status, %w[Interested Applied Interviewed Offered Rejected], :prompt => 'Status' %>
```

Comment:
I can see that we've duplicated this list of statuses now. I can imagine us doing this more in the future on other pages, and then it being tricky to change when we add/remove a status

Can you think of a way to avoid this duplication?